tfhe/src/integer/server_key/radix_parallel/ilog2.rs

```diff
@@ -782,4 +1275,266 @@ pub(crate) mod tests_signed {
             ServerKey::leading_ones,
         );
     }
+
+    pub(crate) fn default_ilog2_test<P>(param: P)
+    where
```

Comment:
In general we could since the idea of signed/unsigned test is the same with the differences being the modulus, clear and ciphertext types, but it would require some work:

-> mapping between unsigned radix and that fact that we need u64 and a certains modulus, same for signed
-> some kind of  struct + trait to have genericity over the `add_under_unsigned/signed_modulus`

Reply:
ok, I think at some point it would be worth it because having 2x the code to maintain is a pain, even if it means having some more complicated trait bounds

Reply:
Yes we need to be able to speend so time on improving the tests